city_metrix/layers/worldpop_age_sex_class.py

```diff
@@ -0,0 +1,33 @@
+from dask.diagnostics import ProgressBar
+import xarray as xr
+import xee
+import ee
+
+from .layer import Layer, get_utm_zone_epsg, get_image_collection
+
+
+class WorldPop(Layer):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+		self.agesex_classes = agesex_classes
+
+    def get_data(self, bbox):
+        # load population
+        dataset = ee.ImageCollection('WorldPop/GP/100m/pop_age_sex')
+        world_pop = ee.ImageCollection(dataset
+                     .filterBounds(ee.Geometry.BBox(*bbox))
+                     .filter(ee.Filter.inList('year', [2020]))
```

Comment:
`year` would be more flexible if added as an input parameter with default value set to 2020

Reply:
added year as input parameter with default 2020